lib/engine/g_1870/stock_market.rb

```diff
@@ -5,13 +5,45 @@
 module Engine
   module G1870
     class StockMarket < StockMarket
+      ISSUE_PAR_PRICES = {
```

Comment:
Comment on what these numbers mean, are they coordinates?

Reply:
Yes, they are coordinates on the stock market where the new par price might be

Reply:
Yeah, a comment is required I think